server/meca/export.test.js

```diff
@@ -50,6 +51,20 @@ describe('MECA integration test', () => {
   })
 
   describe('when generating an archive', () => {
+    it('manifest file is good', async () => {
+      await mecaExport(sampleManuscript, file => 'This is a test')
+
+      const finalName = `${sampleManuscript.id}${mecaPostfix}`
+      const zip = await JsZip.loadAsync(
+        sftp.mockFs.readFileSync(`/test/${finalName}`),
+      )
+
+      const manifest = await zip.files['manifest.xml'].async('string')
+      const manifestJson = JSON.parse(xml2json.toJson(manifest))
+
+      expect(manifestJson).toMatchSnapshot()
+    })
+
```

Comment:
We already have unit tests for the manifest generator [here](https://github.com/elifesciences/elife-xpub/blob/develop/server/meca/file-generators/manifest.test.js). Does this add any more coverage than the existing unit tests?

Reply:
If it does, it may be worth moving the coverage to the generator tests mentioned above.

Reply:
Good point - those tests (obviously) don't work! I'm going to be looking at them and sort them out if its reasonably straight forward as part of this PR.

Reply:
Been thinking about these tests - and these tests do things at a lower layer with the `manuscript.files` objects. This is distinctly different to the tests in export that test the entire export process with the `manuscript` object.